mmdet/apis/ote/apis/detection/ote_utils.py

```diff
@@ -102,6 +103,21 @@ def on_train_batch_end(self, batch, logs=None):
         super().on_train_batch_end(batch, logs)
         self.update_progress_callback(self.get_progress())
 
+    def on_epoch_end(self, epoch, logs=None):
+        self.past_epoch_duration.append(time.time() - self.start_epoch_time)
+        self.__calculate_average_epoch()
+        score = None
+        if hasattr(self.update_progress_callback, 'metric') and isinstance(logs, dict):
+            score = logs.get(self.update_progress_callback.metric, None)
+        self.update_progress_callback(self.get_progress(), score)
```

Comment:
```suggestion
        self.update_progress_callback(self.get_progress(), score=score)
```
This is a workaround for the bug outside of OTE. Please pass `score` as a named argument here.